docker/cortx-build/Makefile

```diff
@@ -96,7 +96,6 @@ _cortx-motr_prereq:
 		sed -i "/BuildRequires.*%{lustre_devel}/d" cortx-motr.spec && \
 		sed -i 's/@BUILD_DEPEND_LIBFAB@//g' cortx-motr.spec && \
 		sed -i 's/@.*@/111/g' cortx-motr.spec && \
-		yum install --setopt=cortx-build-dependencies.module_hotfixes=true --disablerepo='appstream' castxml -y && \
```

Comment:
this is not required?

Reply:
We added this fix due to an issue in EPEL - https://bugzilla.redhat.com/bugzilla/show_bug.cgi?id=2087619. EPEL has fixed this issue now so need to correct it.

Reply:
okay